recipe/meta.yaml

```diff
@@ -34,35 +31,42 @@ requirements:
     - {{ compiler('cxx') }}
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
-    - patch     # [not win]
-    - m2-patch  # [win]
+
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.38.*
+    - llvmlite 0.39.*
     - numpy
```

Comment:
I think numpy should have the same restrictions in host and run:
numpy >=1.18,!=1.22.0,!=1.22.1,!=1.22.2,<1.23
See: https://github.com/numba/numba/blob/0.56.3/setup.py#L373

Reply:
But if in `host` will be `numpy >=1.18,!=1.22.0,!=1.22.1,!=1.22.2,<1.23` then in `run` will be the **latest** numpy version and not the **oldest**.

Reply:
Right, because of numpy's run export.
In that case numpy version should be set to the lowest supported: numpy 1.18

Reply:
To get this one working was quite a bit more complicated. I can see Serhii needed to do this before, so he might remember that `numpy` is tricky :)

I am using a `numpy_min_ver` variable to set the appropriate minimum version, depending on platform and Python version.

See the following commit: https://github.com/AnacondaRecipes/numba-feedstock/commit/d3b3890c29fe8fc959f7da3021e3ba94100d59b0